pom.xml

```diff
@@ -130,7 +130,8 @@
     <freebuilder.version>1.14.9</freebuilder.version>
     <google.code.version>3.0.2</google.code.version>
     <google.errorprone.version>2.1.2</google.errorprone.version>
-    <grpc.version>1.18.0</grpc.version>
+    <grpc.version>1.35.0</grpc.version>
+    <tomcat-annotations.version>1.35.0</tomcat-annotations.version>
```

Comment:
I guess that the `tomcat-annotations.version` is not correct. What is the exact reason for introducing this dependency?
It looks like the dependency isn't maintained, updated last in 2017: https://search.maven.org/artifact/org.apache.tomcat/annotations-api

Reply:
This is resolved.